internal/testcafe/config.go

```diff
@@ -48,6 +49,7 @@ type Suite struct {
 	DisablePageCaching bool              `yaml:"disablePageCaching,omitempty" json:"disablePageCaching"`
 	DisableScreenshots bool              `yaml:"disableScreenshots,omitempty" json:"disableScreenshots"`
 	DisableVideo       bool              `yaml:"disableVideo,omitempty" json:"disableVideo"` // This field is for sauce, not for native testcafe config.
+	Mode               string            `yaml:"mode,omitempty" json:"mode"`
```

Comment:
Hm. Inconsistent ⚠️
For playwright's suite you omitted the json serialization: `json:"-"`

Reply:
🤔 Let me unify them.